doc/dev/developer_guide/testing_integration_tests/tests-integration-testing-teuthology-workflow.rst

```diff
@@ -68,37 +70,45 @@ After building is complete, proceed to trigger tests -
         -e foo@gmail.com \
         -R fail
 
-   Following are the options used in above command with their meanings -
+   The options in the above command are defined here: 
+
+      =============  =========================================================
+         Option        Meaning
+      =============  =========================================================
         -v            verbose
         -m            machine name
-        -c            branch name, the branch that was pushed on ceph-ci
+        -c            the name of the branch that was pushed on ceph-ci
         -s            test-suite name
-        -p            higher the number, lower the priority of the job
-        --filter      filter tests in given suite that needs to run, the arg to
-                      filter should be the test you want to run
-        -e <email>    When tests finish or time out, send an email
-                      here. May also be specified in ~/.teuthology.yaml
-                      as 'results_email'
+        -p            the higher the number, the lower the priority of the job
+        --filter      filter tests in a given suite. the 
+                      arg to filter specifies the test you want to run
+        -e <email>    When tests finish or time out, send an email to the
+                      specified address. Can also be specified in 
+                      ~/.teuthology.yaml as 'results_email'
         -R            A comma-separated list of statuses to be used
-                      with --rerun. Supported statuses are: 'dead',
+                      with --rerun. Supported statuses: 'dead',
                       'fail', 'pass', 'queued', 'running', 'waiting'
                       [default: fail,dead]
+      =============  =========================================================
+
+   .. note:: The priority number present in the command above is a placeholder. 
+      Do not use it in your own tests. See `Testing Priority`_ for information 
+      about priority numbers.
+
+   .. note:: Do not issue a command without a priority number. The default 
+      value is 1000, a value so large that your job is unlikely ever to run.
+
+   Other frequently used/useful options: ``-d`` (or ``--distro``), 
+   ``--distroversion``, ``--filter-out``, ``--timeout``, ``flavor``, 
+   ``-rerun``, ``-l`` (to limit the number of jobs) , ``-n`` (to see the 
+   number of times the job is set to run). Run ``teuthology-suite --help`` 
+   to read descriptions of these and other available options.
 
 #. Wait for the tests to run. ``teuthology-suite`` prints a link to the
-   `Pulpito`_ page created for the tests triggered.
+   `Pulpito`_ page that is created for the tests you have run.
```

Comment:
`Pulpito`_ , where the test results can be viewed.

Reply:
You make it sound like a resort in Mexico.

I've changed the text so that it reads the way you wrote it.